media-plugins/distrho-ports/distrho-ports-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# requires proaudio overlay
+
+EAPI=5
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+
+SRC_URI=""
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
Since this is a live ebuild, it should probably have to keywords so it has to be explicitly unmasked. However, since this project seems to have no releases, maybe it's okay to have this as ~amd64 for now. Not sure, just commenting :)